docker-compose.yml

```diff
@@ -77,7 +79,7 @@ services:
       - nlpsandbox
       - nlpsandbox-internal
     ports:
-      - "80:80"
+      - "${SERVER_PORT}:80"
```

Comment:
@tschaffter This line seems to be creating a bug for me:

```
WARNING: The SERVER_PORT variable is not set. Defaulting to a blank string.
ERROR: The Compose file './docker-compose.yml' is invalid because:
services.nginx.ports contains an invalid type, it should be a number, or an object
```

Reply:
When I called `docker-compose up --build`